test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -27,19 +28,49 @@ describe('OpenxAdapter', () => {
       'auctionId': '1d1a030790a475',
     };
 
-    it('should return true when required params found', () => {
+    let bid_video = {
+      'bidder': 'openxvideo',
+      'params': {
+        'unit': '12345678',
+        'delDomain': 'test-del-domain',
+        'video': {
+          'be': 'true',
+          'url': 'abc.com',
+          'vtest': '1'
+        }
+      },
+      'adUnitCode': 'adunit-code',
+      'sizes': [640, 480],
+      'bidId': '30b31c1838de1e',
+      'bidderRequestId': '22edbae2733bf6',
+      'auctionId': '1d1a030790a475',
+      'transactionId': '4008d88a-8137-410b-aa35-fbfdabcb478e'
+    };
+
+    it('bid: should return true when required params found', () => {
       expect(spec.isBidRequestValid(bid)).to.equal(true);
     });
 
-    it('should return false when required params are not passed', () => {
+    it('bid: should return false when required params are not passed', () => {
```

Comment:
`bid: should return false when required params are not passed` 
should be changed to be easier to read
`should return false when required params are not passed for a banner ad`